sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/tfrecord/TFRecordIOIT.java

```diff
@@ -125,18 +125,19 @@ public void writeThenReadAll() {
         .apply("Transform strings to bytes", MapElements.via(new StringToByteArray()))
         .apply(
             "Record time before writing",
-            ParDo.of(new TimeMonitor<>(TFRECORD_NAMESPACE, "writeTime")))
+            ParDo.of(new TimeMonitor<>(TFRECORD_NAMESPACE, "write_time")))
```

Comment:
What do you think about extracting the string metric names to constants? In case of a change it would be easier to manage the naming.

Reply:
Definitely, thanks!